### PR TITLE
Address Validator dashboard QA issues

### PR DESCRIPTION
**Pull request by eshark9312**
[Validator dashboard QA #4743](https://github.com/Joystream/pioneer/issues/4743)

### CODE REVIEW

**Review by thesan — CHANGES_REQUESTED** [review]
I added a commit related to the `ExternalResourceLink` constant.

But also please add 2 decimals to the aprs

File: packages/ui/src/memberships/constants/externalResourceLink.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const ExternalResourceLink = {
+  TELEGRAM: 'https://web.telegram.org/k/#@',
```
Comment (thesan) [inline packages/ui/src/memberships/constants/externalResourceLink.ts:2]: I don't know much about Telegram but @ivanturlakov suggested to change this to:
```suggestion
  TELEGRAM: 'https://t.me/',
```

**Comment by kdembler** [comment]
There's this intermediate loading state, where we show some weird stats like:
```
Validator 43/-43
Nominator 0/329
```

Can we postpone showing this data until we have the full stats?

https://github.com/Joystream/pioneer/assets/12646744/6c78e779-fc7f-4d2a-80fd-fb28c7eb6aad

File: packages/ui/src/validators/components/ValidatorsList.tsx [hunk 2]
```diff
@@ -65,7 +65,7 @@ export const ValidatorsList = ({ validators, eraIndex, order, pagination }: Vali
                     The APR is subject to the amount staked and have a diminishing return for higher token amounts. This
                     is calculated as follow:
                     <br />
-                    <code>Yearly Reward * Commission / Stake</code>
+                    <code>Yearly Reward * (1 - Commission) / Stake</code>
```
Comment (thesan) [inline packages/ui/src/validators/components/ValidatorsList.tsx:68]: Are you sure about that ? It should show the nominator reward not the validator one. I might be wrong though based on @ivanturlakov's comments but could you detail why it should be `Reward * (1 - Commission) / Stake` please ?

**Review by thesan — CHANGES_REQUESTED** [review]

File: packages/ui/src/validators/components/ValidatorsList.tsx (hunk 2 above)
Comment (eshark9312) [reply packages/ui/src/validators/components/ValidatorsList.tsx:68]: I have checked, please check here.
https://handbook.joystream.org/system/validation

Comment (eshark9312) [reply packages/ui/src/validators/components/ValidatorsList.tsx:68]: https://handbook.joystream.org/system/validation#calculating-rewards

File: packages/ui/src/validators/hooks/useValidatorsList.tsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ export const useValidatorsList = () => {
       key,
       isDescending: key !== state.key ? DESCENDING_KEYS.includes(key) : !state.isDescending,
     }),
-    { key: 'default', isDescending: false }
+    { key: 'commission', isDescending: false }
```
Comment (thesan) [inline packages/ui/src/validators/hooks/useValidatorsList.tsx:20]: Sorry I didn't meant it like that just that... just to sort in ascending order once click the user clicks on it
```suggestion
    { key: 'default', isDescending: false }
```

**Review by thesan — CHANGES_REQUESTED** [review]

**Review by thesan — APPROVED** [review]
LGTM!

There's still some talks on Discord about APRs/APYs but I think it's best to merge this now. If some follow ups are needed it should be done in another PR.

Nice work :+1:

**Pull request merged**